tests/MongoDB.Driver.Core.Tests/Core/Operations/ReIndexOperationTests.cs

```diff
@@ -25,6 +25,7 @@
 
 namespace MongoDB.Driver.Core.Operations
 {
+    [Obsolete("This class will be removed in a later version of the driver.")]
```

Comment:
Do we really need to mark this test class as obsolete? :)

Reply:
Do we need to? Not necessarily. 

But it's easier than using a bunch of `#pragma warning disable 618` lines.

I'm happy to not mark it obsolete if you want.

Reply:
maybe a better option is adding a global `#pragma warning` for the whole file?

Reply:
done